paddle/fluid/operators/set_value_op.h

```diff
@@ -59,26 +59,70 @@ inline std::string GetValueName(framework::proto::VarType::Type data_type) {
   return value_name;
 }
 
-inline framework::DDim GetSliceDims(const framework::DDim in_dims,
-                                    const std::vector<int64_t> axes,
-                                    const std::vector<int64_t> starts,
-                                    const std::vector<int64_t> ends) {
-  framework::DDim slice_dims(in_dims);
-
+inline void CheckAndUpdateSlice(const framework::DDim in_dims,
+                                const std::vector<int64_t> axes,
+                                int64_t* starts, int64_t* ends,
```

Comment:
Prefer to use `std::vector<int64_t>*` than `int64_t*`

Because you are operating `data()` of a vector. It is OK now, but in the future what if some other software make the pointer break some property of the `vector`.

Using use `std::vector<int64_t>*` is more OOP here.

Reply:
Thanks, done.